docs/HowToFAQ/FAQNoBody.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Is there a way to prevent Warning: note, this forall statement has no body  from occurring? I have a forall loop with no body that results in the lemma verifying, but if I add a body (even an empty body) the lemma doesn't verify.
```

Comment:
```suggestion
title: Is there a way to prevent "Warning: note, this forall statement has no body" from occurring? I have a forall loop with no body that results in the lemma verifying, but if I add a body (even an empty body) the lemma doesn't verify.
```